app-misc/cmatrix/cmatrix-2.0-r5.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+FONT_SUFFIX="pcf"
+
+inherit cmake font
+
+DESCRIPTION="Ncurses based app to show a scrolling screen from the Matrix"
+HOMEPAGE="https://github.com/abishekvashok/cmatrix"
```

Comment:
The homepage URL you dropped,  ttps://www.asty.org/cmatrix/, seems to be still valid. Why drop it?

Reply:
The website used to be valid but is no longer maintained:

> The current version of CMatrix is 1.2a. 

The current HOMEPAGE is now up to date.

Reply:
Alright, according to https://www.asty.org/cmatrix/,

> You can find the newest repository for CMatrix at https://github.com/abishekvashok/cmatrix and look for a new official website as well!

it seems sensible to drop  https://www.asty.org/cmatrix/.